data/baibhav-tiwari.json

```diff
@@ -18,8 +26,52 @@
     },
     {
       "name": "Linkedin",
-      "url": "https://www.linkedin.com/in/baibhavtiwari/",
+      "url": "https://www.linkedin.com/in/baibhavtiwari",
       "icon": "FaLinkedin"
+    },
+    {
+      "name": "Hashnode",
+      "url": "https://hashnode.com/@baibhavv",
+      "icon": "SiHashnode"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Learning Cpp",
+      "date": "2021",
+      "icon": "FaSearch",
+      "color": "red",
+      "description": "It was my first programming language which I started Learning."
+    },
+    {
+      "title": "Started Learning Web development",
+      "date": "2022",
+      "icon": "FaGlobe",
+      "color": "green",
+      "description": "i was always interested in creating websites, so i decided to learn web dev."
+    },
+    {
+      "title": "Social Media",
+      "date": "2022",
+      "icon": "FaAndroid",
+      "color": "green",
+      "description": "Firstly i thought Twitter is boring but as i started using it more i started loving it.",
+    
+    },
+    {
+      "title": "My tech stack/ what i know",
+      "date": "2022",
+      "icon": "FaGraduationCap",
+      "color": "green",
+      "description": "HTML, CSS, JavaScript, Tailwind, Nodejs, Expressjs, React, Python, Cpp",
+    },
+    {
+      "title": "Open-Source",
+      "date": "2022",
+      "icon": "DiOpensource",
+      "color": "#000",
+      "description": "It's been 3 Months  and more that I'm contributing to other's project and doing Open-Source",
+      
     }
   ],
   "milestones": [
```

Comment:
You added duplicate Milestones code. Remove one. You can check the full profile example here: https://linkfree.eddiehub.io/docs/full-profile-example